src/js/common/components/form-elements/GrowableTable.jsx

```diff
@@ -212,12 +248,22 @@ GrowableTable.propTypes = {
   isValidSection: React.PropTypes.func.isRequired,
   addNewMessage: React.PropTypes.string,
   minimumRows: React.PropTypes.number,
-  rowTitle: React.PropTypes.string
+  rowTitle: React.PropTypes.string,
```

Comment:
Pretty sure HCA now uses the common `GrowableTable` component, so these changes may also affect HCA. Need to make sure everything works there as well.